__tests__/api.test.js

```diff
@@ -72,6 +70,30 @@ describe('GET - /api/articles/:article_id', () => {
   });
 })
 

```

Comment:
Need a few more tests and error handling for this patch request, we'll need to check
- 400 invalid article id
- 404 valid article id but doesn't exist
- 400 missing or incorrect body (like if inc_votes is missing or incorrect)